packages/paste-website/src/pages/components/separator/index.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Separator - Components
+description: The Separator component can be used to create visual and semantic separation between two or more UI controls.
```

Comment:
What's "semantic separation"?

Reply:
prob don't need "two or more"

Reply:
And maybe "elements" instead of "controls"? Like people could use separators to divide static text.

Reply:
Semantic is the machine readable, non-visual communication of separation.

Reply:
Is this because screenreaders will read out where separators show up on a page?

Reply:
Not necessarily where, but what they are. When you are navigating a page you'll come across this thing which will announce itself as a separator. So you'll now know we've created separation, a grouping, or what ever without having to see a line